internal/attestation/crafter/runners/jenkinsjob.go

```diff
@@ -0,0 +1,65 @@
+//
+// Copyright 2023 The Chainloop Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package runners
+
+import "os"
+
+type JenkinsJob struct{}
```

Comment:
This is not related to your change, since the other runners have the same issue, but it's something that we could tackle in another patch.

It seems that the "factory functions" are not being used outside of tests, when they could also be used in `runner.go` in the switch clause.